.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
     - node_modules
 env:
   matrix:
-  - EMBER_TRY_SCENARIO=ember-1-12
+  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
I did not realize we were specifically staying with the other - this was an Ember-CLI upgrade change.

I have reverted it.